spec/features/bulk_updates/bulk_updates_science_keywords_spec.rb

```diff
@@ -390,11 +396,13 @@
         end
 
         it 'displays the updated keywords' do
-          within '.science-keywords-preview' do
-            expect(page).to have_no_content('EARTH SCIENCE ATMOSPHERIC TEMPERATURE SURFACE TEMPERATURE', normalize_ws: true)
+          within '#metadata-preview' do
+            expect(page).to have_no_content('SURFACE TEMPERATURE', normalize_ws: true)
             expect(page).to have_no_content('EARTH SCIENCE SOLID EARTH ROCKS/MINERALS/CRYSTALS SEDIMENTARY ROCKS', normalize_ws: true)
```

Comment:
I think this expect can just be SEDIMENTARY ROCKS, you should remove the rest in that string